src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.Windows.cs

```diff
@@ -252,5 +260,34 @@ internal int GetFileType()
 
             return fileType;
         }
+
+        internal unsafe long GetFileLength()
```

Comment:
Nit: could the `unsafe` be moved to the local function?

Reply:
I'll address it in a PR with SetLength